src/components/job-common.js

```diff
@@ -2,76 +2,120 @@ import { Fragment } from 'react'
 import { div, h, h3, h4 } from 'react-hyperscript-helpers'
 import { Link } from 'src/components/common'
 import { icon } from 'src/components/icons'
+import { TooltipCell } from 'src/components/table'
 import colors from 'src/libs/colors'
 import * as Nav from 'src/libs/nav'
 import * as Style from 'src/libs/style'
 
 
-export const collapseStatus = status => {
-  switch (status) {
+const iconSize = 24
+export const addCountSuffix = (label, count = undefined) => {
+  return label + (count === undefined ? '' : `: ${count}`)
+}
+
+export const statusType = {
+  succeeded: {
+    id: 'succeeded',
+    label: _ => 'Succeeded',
+    icon: style => icon('check', { size: iconSize, style: { color: colors.success(), ...style } })
```

Comment:
Note that I removed the aria-labels from the icons. You only want to put an aria-label on an icon when the information represented by the icon is not otherwise available; otherwise, you want to mark it aria-hidden (which our icon class does if a label is not present). In all the places where we show these icons, we represent the icon status through a label or through summary text, so the aria-label was just adding extra noise when navigating through the page.

Example locations:
![image](https://user-images.githubusercontent.com/484484/151879949-821e7a9a-7968-4027-ade0-f6f2e3ff230a.png)
![image](https://user-images.githubusercontent.com/484484/151880002-2a7b02f8-8c1d-4e80-bcb6-913521db23cd.png)

![image](https://user-images.githubusercontent.com/484484/151880028-b11e7189-8dc9-49bc-9f7d-eccc8e452a5d.png)

The only tricky place is the one below, and the information being displayed in the popup is already being summarized on the cell entry via an aria-label.

![image](https://user-images.githubusercontent.com/484484/151880151-43ab65d8-9c8d-44ca-8d95-9136af479ee1.png)

![image](https://user-images.githubusercontent.com/484484/151880229-a1e4401a-ef9e-4cb2-8919-771696640df7.png)